src/libstore/include/nix/store/filetransfer.hh

```diff
@@ -102,6 +102,7 @@ struct FileTransferRequest
     bool verifyTLS = true;
     bool head = false;
     bool post = false;
+    bool delete_ = false;
```

Comment:
Feels like it might be time to combine `post` and `delete_` into an `enum struct` for the HTTP methods.

Reply:
Done!